requirements.txt

```diff
@@ -23,6 +27,7 @@ python-dateutil==2.8.2
 python-decouple==3.8
 python-dotenv==1.0.0
 pytz==2023.3.post1
+requests==2.31.0
 s3transfer==0.8.2
 six==1.16.0
 sqlparse==0.4.4
```

Comment:
Thanks for getting this updated so quickly, and great job locating the issue so fast. Appreciate you!